source/common/http/http1/conn_pool.h

```diff
@@ -139,8 +141,9 @@ class ConnPoolImpl : Logger::Loggable<Logger::Id::pool>, public ConnectionPool::
 class ConnPoolImplProd : public ConnPoolImpl {
 public:
   ConnPoolImplProd(Event::Dispatcher& dispatcher, Upstream::HostConstSharedPtr host,
-                   Upstream::ResourcePriority priority)
-      : ConnPoolImpl(dispatcher, host, priority) {}
+                   Upstream::ResourcePriority priority,
+                   const Network::ConnectionSocket::OptionsSharedPtr& options = nullptr)
```

Comment:
nit: default param here seems not worth it.

Reply:
I thought I had all of these removed, checked and this was the last one. Thanks!